patroni/dcs/__init__.py

```diff
@@ -1698,6 +1698,16 @@ def reset_cluster(self) -> None:
             self._cluster = None
             self._cluster_valid_till = 0
 
+    def ensure_cluster_access(self, sleep_time: int = 5) -> Optional[Cluster]:
+        """Continuously attempt to retrieve cluster from DCS, retrying on :exc:`DCSError` with delay."""
```

Comment:
The docstring must include method arguments and returned value.
Please take other methods as an example.